source/app.js

```diff
@@ -30,6 +31,35 @@ function getCurrentRouteName(navigationState: NavigationState): ?string {
 type Props = {}
 
 export default class App extends React.Component<Props> {
+	componentDidMount() {
+		startStatusBarColorChanger()
```

Comment:
Where is the start/stop coming from? Currently it breaks things.

Reply:
I feel like it might be related to the way we were telling the iOS status bar to be dark or light? @hawkrives